builders/xcm/remote-evm-calls.md

```diff
@@ -0,0 +1,337 @@
+
+Because a remote EVM call does not have the actual `v-r-s` values of the signature, there could be collusion problems of the EVM transaction hash, as it is calculated as the Keccak256 hash of the signed transaction blob. Consequently, if two accounts with the same nonce submit the same transaction object, they will end up with the same EVM transaction hash. Therefore, all remote EVM transactions use a global nonce that is attached to the [Ethereum-XCM pallet](https://github.com/PureStake/moonbeam/tree/master/pallets/ethereum-xcm){target=_blank}.
+
+Another significant difference is in terms of the gas price. The fee for remote EVM calls is charged at an XCM execution level. Consequently, the EVM gas price is zero, and the EVM will not charge for the execution itself. This can also be seen in the receipt of a remote EVM call transaction.
+
+Lastly, a lower gas limit is available for remote EVM calls. Currently, an XCM message can only take `20,000,000,000` weight (that is, `0.02` seconds of block execution time). For all Moonbeam-based networks, there is a ratio of [`25,000` units of gas per unit of weight](https://github.com/PureStake/moonbeam/blob/master/runtime/moonbase/src/lib.rs#L371-L375){target=_blank}. Considering that you need some of the XCM message weight limit to execute the XCM instructions themselves, a remote EVM call might have around `18,000,000,000` weight left, which is `720,000` gas units. Consequently, the maximum gas limit you can provide for a remote EVM call is around `720,000` gas units. Note that this might change in the future.
+
+In summary, these are the main differences between regular and remote EVM calls:
+
+1. Remote EVM calls use a global nonce (owned by the [Ethereum-XCM pallet](https://github.com/PureStake/moonbeam/tree/master/pallets/ethereum-xcm){target=_blank}) instead of a nonce per account
+2. The `v-r-s` values of the signature for remote EVM calls are `0x1`. The sender can't be retrieved from the signature through standard methods (for example, through [ECRECOVER](/builders/pallets-precompiles/precompiles/eth-mainnet/#verify-signatures-with-ecrecover){target=_blank}). Nevertheless, the `from` is included in both the transaction receipt and when getting the transaction by hash (using the Eth JSON RPC)
+3. The gas price for all remote EVM calls is zero. The EVM execution is charged at an XCM execution level and not at an EVM level
+4. The maximum gas limit you can set for a remote EVM call is `720,000` gas units
+
+## Ethereum-XCM Pallet Interface {: #ethereum-xcm-pallet-interface}
```

Comment:
This section should probably be linked in the "Pallets" section under "Pallets & Precompiles"

Reply:
@eshaben we could do a pallets section only for `EthereumXCM`? What do you think?